label_studio_converter/converter.py

```diff
@@ -422,7 +442,8 @@ def convert_to_coco(self, input_data, output_dir, output_image_dir=None, is_dir=
                         'bbox': [x, y, w, h],
                         'ignore': 0,
                         'iscrowd': 0,
-                        'area': w * h
+                        'area': w * h,
+                        'annotator': item['completed_by'].get('email')
```

Comment:
I'm not sure this format is supported by COCO - there is no `"annotator"` key [here](https://cocodataset.org/#format-data)